protocol/dubbo/codec.go

```diff
@@ -88,10 +92,12 @@ func (p *DubboPackage) Unmarshal(buf *bytes.Buffer, opts ...interface{}) error {
 			return perrors.Errorf("opts[0] is not of type *Client")
 		}
 
-		p.Body = client.GetPendingResponse(SequenceType(p.Header.ID)).reply
-		if p.Body == nil {
+		pendingRsp := client.GetPendingResponse(SequenceType(p.Header.ID))
+		if pendingRsp == nil {
 			return perrors.Errorf("client.GetPendingResponse(%v) = nil", p.Header.ID)
 		}
+		response := &hessian.Response{RspObj: pendingRsp.reply}
+		p.Body = response
```

Comment:
to avoid a temple var, just a assignment may be better

Reply:
ok， a temple var should be deleted.